paper-styles.html

```diff
@@ -30,6 +30,20 @@
 We recommend importing each of these individual files, and using the style mixins
 available in each ones, rather than the aggregated `paper-styles.html` as a whole.
 
+Create your own global theme to override components that use paper-styles by creating a component file as follows and 
+then import it into your index.html or app component using html imports like you do with other web components.
```

Comment:
Sooo the _app-component_ isn't really right. For that, you would want a shared module (https://www.polymer-project.org/1.0/docs/devguide/styling#style-modules), so that your styles are actually scoped.